tests/db_functions/datetime/test_now.py

```diff
@@ -47,3 +49,18 @@ def test_basic(self):
             ["How to Time Travel"],
             lambda a: a.title,
         )
+
+    def test_microseconds(self):
+        Article.objects.create(
+            title="How to Django",
+            text=lorem_ipsum,
+            written=timezone.now(),
+        )
+
+        now_string = (
+            Article.objects.annotate(now_string=Cast(Now(), TextField()))
+            .get()
+            .now_string
+        )
+        precision = connection.features.time_cast_precision
+        self.assertRegex(now_string, rf"^.*\.\d{{{precision}}}")
```

Comment:
Missed whimsy: if Black didn't normalize it to 'rf', this could be a 'fr', or French string.

Reply:
Trailing zeros are stripped on PostgreSQL, so I changed to 
```suggestion
        self.assertRegex(now_string, rf"^.*\.\d{1, {{precision}}}")
```